files/apt/sources.list.j2

```diff
@@ -1,20 +1,26 @@
 # The configuration is generated by template
 # Please add additional sources in /etc/apt/sources.list.d
 
+{% if DISTRIBUTION == 'bookworm' -%}
+{%- set nonfree_component='non-free-firmware' -%}
```

Comment:
For Bookworm, `non-free-firmware` is definitely needed. I don't _think_ `non-free` is needed because there aren't any non-free software we install from the Debian repos.